### PR TITLE
Individuals restore changes

### PR DESCRIPTION
**Pull request by almanilsson**
Added a button which restores the sample set IDs to the original population IDs

### CODE REVIEW

**Review by percyfal — COMMENTED** [review]
So I'm still getting weird zoom-in effects, both in firefox and chrome. Also, in the dropdown list for New sample set ID I get all kinds of choices. Don't know if this is related to this PR or not.

screen

**Comment by percyfal** [comment]
I see the sample set dropdown in another PR, could be my browser cache.

**Comment by almanilsson** [comment]
> So I'm still getting weird zoom-in effects, both in firefox and chrome. Also, in the dropdown list for New sample set ID I get all kinds of choices. Don't know if this is related to this PR or not.
> 
> screen

The map looking strange is due to the use of the development environment. If you run it using 
`rye run python -m tseda serve tests/data/test.trees.tseda`
instead then this is not a problem. 

The dropdown thing is indeed your cache

**Comment by percyfal** [comment]
Indeed it is the development environment. Interesting! I need to be get home now, I will address the remaining comments as soon as I can.

File: src/tseda/datastore.py [hunk 1]
```diff
@@ -92,6 +92,7 @@ class IndividualsTable(Viewer):
         description=("Reassign individuals to this sample set ID."),
     )
     mod_update_button = pn.widgets.Button(name="Update")
+    restore_button = pn.widgets.Button(name="Restore", button_type="danger")
```
Comment (percyfal) [inline src/tseda/datastore.py:95]: One could possibly add a tooltip that explains what the button does.

**Review by percyfal — APPROVED** [review]
Nice addition!

**Pull request merged**